docs/getting-started/install-resoto/kubernetes.md

```diff
@@ -123,19 +123,19 @@ resotoworker:
 
 ### Install Helm Chart
 
-Clone the [`someengineering/resoto`](https://github.com/someengineering/resoto) repository:
+Add the resoto helm repository:
 
 ```bash
-$ git clone https://github.com/someengineering/resoto
+$ helm repo add resoto https://helm.resoto.com
 ```
 
 Next, install Resoto using Helm:
 
 ```bash
-$ helm install resoto ./resoto/kubernetes/chart --set image.tag={{latestRelease}} -f resoto-values.yaml
+$ helm install resoto resoto/resoto -f <values.yaml>
 ```
 
-And just like that, you have Resoto running in Kubernetes! A collect run will begin automatically. This first collect usually takes less than 3 minutes.
+And just like that, you have Resoto running in your Kubernetes cluster! A collect run will begin automatically. This first collect usually takes less than 3 minutes.
```

Comment:
```suggestion
And just like that, you have Resoto running in a Kubernetes cluster! A collect run will begin automatically. This first collect usually takes less than 3 minutes.
```

Reply:
Most of the language of article is directed to a user perspective, and last making it `your cluster` was more appropriate slang.

Reply:
"Your" suggests that a user only has a single Kubernetes cluster, or that they themselves own it. "A" is more appropriate here.